.gitignore

```diff
@@ -160,4 +160,8 @@ crates/rolldown_binding/artifacts/
 packages/**/THIRD-PARTY-LICENSE
 
 # Opinionated configuration
-.vscode/launch.json
\ No newline at end of file
+.vscode/launch.json
+
+# node_modules in tests
+!/crates/rolldown/tests/**/node_modules
```

Comment:
I'm not sure. We might rely on `node_modules` that created by `npm`, we just don't have the case currently.

Reply:
In the future, we probably will ensure the correctness of mainstream libraries, such as vue, react and antd etc. It's better we install them by `pnpm` instead of having a copy in rolldown's repo.

Reply:
what about 
`!/crates/rolldown/tests/**/esbuild/node_modules`, limit the scope under **esbuild** tests.

Reply:
Yeah. That should work.